client/src/main/java/io/split/storages/pluggable/adapters/UserCustomSplitAdapterProducer.java

```diff
@@ -98,6 +99,13 @@ public void decreaseTrafficType(String trafficType) {
         }
     }
 
+    @Override
+    public void updateFeatureFlag(ParsedSplit parsedSplit) {
+        List<ParsedSplit> parsedSplits = new ArrayList<>();
+        parsedSplits.add(parsedSplit);
+        putMany(parsedSplits);
```

Comment:
Not sure if you have `Collections` but if it's available..
```suggestion
        putMany(Collections.singletonList(parsedSplit));
```

Reply:
I updated.